src/olympia/api/serializers.py

```diff
@@ -40,9 +41,16 @@ def get_fields(self):
                 fields[key] = ESTranslationSerializerField(source=field.source)
         return fields
 
-    def to_native(self, data):
+    def to_internal_value(self, data):
```

Comment:
`to_native` is equal to `to_representation` and makes sense here - `from_native` and its equal, `to_internal_value` does not.